project/eScavenge/CLI.py

```diff
@@ -289,11 +289,39 @@ def answer_question(self, args):
         return "There are no more landmarks!"
     return "Incorrect Answer! The Question Was: \n{}".format(self.game.get_team_landmark(self.current_user).question)
 
+@need_admin
+def edit_penalty_value(self, args):
+    if self.game.started or self.game.ended:
```

Comment:
Checking for game status should be (and is) done in Game itself. So this ends up being duplicate code essentially.